### ISSUE TITLE
Make it easier for users to know which drains they've adopted

### ISSUE THREAD

**Issue by jasonlally** [sfbrigade/adopt-a-drain#88]
Aside from the icons, we likely need a quick way for users to access their adopted drains via a list or other simple mechanism. Given the 20,000+ drains this seems necessary to help users come back and "manage" their drain adoptions.

Based on email from Jean:
3)	Another tester forgot which drain she adopted and wanted to be able to search for it

**Comment by jszwedko** [comment]
This was something OpenOakland was interested in too.

**Comment by jeanwalshie** [comment]
yes!

**Comment by jeanwalshie** [comment]
can this be an MVP item? i wonder if it's possible to have the user's adopted drains show up no matter  where/how zoomed in they are on the map. or would it be a search? "Find my drains"

**Comment by jszwedko** [comment]
@jeanwalshie what do you think about including the list of drains in-between the `Find Drains` and `Edit Profile` buttons on: 
![2016-01-20-204437_316x250_scrot](https://cloud.githubusercontent.com/assets/316880/12471873/c0db1aa0-bfb6-11e5-922c-84ae7c83598e.png)

I think we can probably do this as MVP if we keep the UI simple (just the list of drains and you can click to zoom the map to the location).

**Comment by jeanwalshie** [comment]
sure @jszwedko, like a button that says, "Find my drains", perhaps using the orange icon color?

**Comment by jszwedko** [comment]
@jeanwalshie I was thinking about just listing them out right there, so it'd be something like:

```
<Find drains button>

Your drains:
Whatever you named the drain <- would be a link to zoom the map to the location
Whatever you named your second drain
Whatever you named your third drain

<Edit profile button>
<Sign out button>
```

What do you think of that?

**Comment by jeanwalshie** [comment]
thanks, @jszwedko -- it sounds good but i think i'd need to see it to fully understand. there wouldn't be a button, just a listing of links to each drain location? i wonder if that would take up too much real estate if folks had adopted several drains? either way, i'd love to try it!

**Comment by jszwedko** [comment]
I was thinking a list of links -- my thought is that users would probably adopt a small enough subset of drains (say <10) that it wouldn't be too much of an issue, but it would be able to scroll, of course. Basically it would push the "edit profile" and "sign out" buttons lower which probably see very little interaction anyway.

I can take a stab and send a screenshot of what I'm thinking.

**Comment by jeanwalshie** [comment]
@jszwedko that sounds great. you're right, i certainly hope people don't adopt more than a few drains!

**Comment by jszwedko** [comment]
Closed by #135

**Comment by jeanwalshie** [comment]
hurray! thanks @jszwedko

**Comment by jeanwalshie** [comment]
Sorry -- one more thing! Can we make the font for the new "My drains" the same size and style as "Address/Neighborhood"? and left justify it? OR create a button (possibly orange to match the color or an adopted drain) and make the font the same as the "Find drains" font and keep it centered?

**Comment by jszwedko** [comment]
Closed by #135

**Comment by jszwedko** [comment]
Oops, missed your last comment -- I'll update the styling.

**Comment by jszwedko** [comment]
Closed by #151

**Comment by jeanwalshie** [comment]
gracias @jszwedko!
